api/mesh/v1alpha1/dataplane.proto

```diff
@@ -16,41 +16,12 @@ message Dataplane {
   option (kuma.mesh.resource).name = "DataplaneResource";
   option (kuma.mesh.resource).type = "Dataplane";
   option (kuma.mesh.resource).package = "mesh";
-  option (kuma.mesh.resource).kds.send_to_zone = true;
   option (kuma.mesh.resource).kds.send_to_global = true;
   option (kuma.mesh.resource).ws.name = "dataplane";
 
   // Networking describes inbound and outbound interfaces of a dataplane.
   message Networking {
 
```

Comment:
```suggestion
reserved 6; // Formerly ingress mode message, see #3435 
```

Just to be safe, we should reserve this field.

Reply:
good point! added